config/routes.rb

```diff
@@ -9,6 +9,10 @@
   # You can have the root of your site routed with "root"
   root 'welcome#index'
 
+  resources :list do
```

Comment:
Any way around this? This actually instantiates default RESTful routes for the `List` controller and removing this block would get rid of those routes.

I could write the routes manually myself, but this seemed really handy. Anything I'm missing here?